README.md

```diff
@@ -1,257 +1,116 @@
-
-[![Known Vulnerabilities](https://snyk.io//test/github/flexion/ef-cms/badge.svg?targetFile=package.json)](https://snyk.io//test/github/flexion/ef-cms?targetFile=package.json)
-
-## Screenshots of new system
-
-### Case detail page / Docket record
-<a href="docs/images/screenshot_docket.png"><img src="docs/images/screenshot_docket.png" width="80%" style="float: left; margin: 0 4px;" /></a>
-
-### Quality control for documents
-<a href="docs/images/screenshot_qc.png"><img src="docs/images/screenshot_qc.png" width="80%" style="float: left; margin: 0 4px;" /></a>
+**Flexion** is the competitively awarded contractor that has been working on DAWSON since 2018. **18F** is a federal digital service team serving as advisors to the U.S. Tx Court and have paired with the U.S. Tax Court since the inception of DAWSON.
 
-### Validating a Petition
+### User types
 
-<a href="docs/images/screenshot_petition.png"><img src="docs/images/screenshot_petition.png" width="80%" style="float: left; margin: 4px 4px 0 0;" /></a>
+- **Petitioners** - Citizens filing a petition with the court; if filing without legal representation, referred to as pro-se petitioner
+- **Practitioners** - Attorneys who file a petition on behalf of a petitioner
+- **Internal court users** - System users responsible for managing cases within the system, work at Tax Court
+- **IRS attorneys** - Work for IRS and file answers to petitions
+- **General public** - Can search for a case on public website and see certain details about it that are open to the public
 
-### Trial session details
+### Project status
 
-<a href="docs/images/screenshot_trialsession.png"><img src="docs/images/screenshot_trialsession.png" width="80%" style="float: left; margin: 4px 0 0 4px;" /></a>
+DAWSON is in active use and is continually updated to deploy enhancements and new features.
 
-<br clear="both">
+- [Release notes](https://www.ustaxcourt.gov/release_notes.html) for DAWSON
+- [How we work](./docs/how-we-work.md) has our principles, product team, technical strategy, meetings cadence, tools, etc.
+- [Onboard](https://github.com/ustaxcourt/ef-cms/issues/new?template=onboarding.md) or [offboard](https://github.com/ustaxcourt/ef-cms/issues/new?template=offboarding.md) a teammate
+- [Documentation](./docs/README.md) about the CI/CD setup, API, style guide, UX, code review, etc.
+- [Ongoing development documentation](./wikiwiki/README.md) such as designs, research data, user workflows, etc.
+- [Product roadmap](https://docs.google.com/document/d/1g3D1zPNQqVsWhJ6uGIVcHA04IdBHgYcOCm9roaSfFNk/edit) 🔒
+- [Goals of DAWSON](https://docs.google.com/document/d/1xCh1hbXjJItOWlQ2MH-GDDc0KWho88KKcHfUJcpHkSs/edit) 🔒
```

Comment:
This document (Goals) is not accessible to the general public, afaict.

Reply:
Correct! They’re referenced here with a lock icon (🔒) to indicate as such — I think keeping the documentation together and linking out to private sections helps keep everyone on the same page. One example of this pattern is the [TTS Handbook](https://handbook.tts.gsa.gov/) which also adds lock icons to indicate when items are private.